packages/postgres-database/src/simcore_postgres_database/models/projects_nodes.py

```diff
@@ -36,42 +48,10 @@
     # TIME STAMPS ----
     column_created_datetime(timezone=True),
     column_modified_datetime(timezone=True),
+    sa.UniqueConstraint(
+        "project_uuid", "node_id", name="projects_nodes__project_uuid_node_id_ukey"
+    ),
```

Comment:
should this not be the primary key at this point? Is this table lacking one?

Reply:
true. good catch thanks.